ISHPermissionKit/Requests/ISHPermissionRequestNotificationsRemote.m

```diff
@@ -0,0 +1,47 @@
+//
+//  ISHPermissionRequestNotificationsRemote.m
+//  ISHPermissionKit
+//
+//  Created by Felix Lamouroux on 01.09.2015.
+//  Copyright (c) 2014 iosphere GmbH. All rights reserved.
+//
+
+#import "ISHPermissionRequestNotificationsRemote.h"
+#import "ISHPermissionRequest+Private.h"
+
+@implementation ISHPermissionRequestNotificationsRemote
+
+#ifdef __IPHONE_8_0
+
+- (ISHPermissionState)permissionState {
+    if (!NSClassFromString(@"UIUserNotificationSettings")) {
```

Comment:
`if (![UIUserNotificationSettings class])` should work fine, without the need for String-based checks.